spec/fixtures/proofing/lexis_nexis/true_id/true_id_response_failure_no_liveness.json

```diff
@@ -15,7 +15,7 @@
       "ProductType": "TrueID",
       "ExecutedStepName": "True_ID_Step",
       "ProductConfigurationName": "AndreV3_TrueID_Flow",
-      "ProductStatus": "pass",
+      "ProductStatus": "fail",
```

Comment:
Can you say more about these changes (also in some of the other fixtures)? I could be mistaken but I thought these were direct responses from TrueID and this PR is to change what field we rely on from TrueID, and not that the TrueID response itself has changed.

Reply:
@night-jellyfish - yes, you are correct ... this change is about relying on a different field (transaction_status)! 👍🏿
while we will not be evaluating product_status ... in our prod events.log, it is consistent that product status and transaction status either both pass or both fail. however, it seems there is a discrepancy in many of our fixtures.

Reply:
Interesting! I guess I was worrying a little that I thought our fixtures were taken directly from requests to the API (and then anonymized). So that would imply to me that maybe the values could be opposing. But maybe I'm wrong about how they were created and this isn't a concern.